test/unit/setup-test.js

```diff
@@ -31,18 +33,25 @@ describe('api.setup', function() {
         ApiSettings.debug = false;
         // remove fixture and player instances
         const fixture = document.querySelector('#test-fixture');
+        if (api) {
+            api.remove();
+        }
         if (fixture.parentNode) {
             fixture.parentNode.removeChild(fixture);
         }
         for (let i = instances.length; i--;) {
             instances[i].remove();
         }
         sandbox.restore();
+        api = null;
```

Comment:
Before these changes, were were not tearing down all api instances (and their views) properly, which left `view` instances in the `viewManager`s list of views.